opensrp-chw-core/src/main/java/org/smartregister/chw/core/activity/CoreAncRegisterActivity.java

```diff
@@ -113,17 +113,6 @@ public void startFormActivity(JSONObject jsonForm) {
         }
     }
 
-    @Override
```

Comment:
Was there a reason to delete these @manutarus? The two are overridden in HF so app won't build when they are deleted.

Reply:
@ellykits, this was to better manage how we get the register activity we should go to especially on the pregnancy outcome form that could take you to pnc, family or remain on anc member profile. We can deprecate the methods but i think you may need the fix on HF as well